pkg/testers/exec/exec.go

```diff
@@ -64,8 +65,23 @@ func (t *Tester) Execute() error {
 	return t.Test()
 }
 
+func expandEnv(args []string) []string {
```

Comment:
usually an expandenv function supports shell syntax like `--foo=${BAR}`.

we should look at how kubernetes implements this for pods.

Reply:
It's using `os.ExpandEnv` inside so this syntax also works?